### DISCUSSION TITLE
zed editor extension support

### DISCUSSION THREAD

**Discussion by Raynos** [oxc-project/oxc#5654] [Feature Request]
I switched our monorepo for our 15 engineers from eslint to oxlint

And a coworker said he's using https://zed.dev/ and would love to have an extension for that editor.

**Comment by IWANABETHATGUY** [comment]
I am interested in this, but I don't have time for now.
for anyone interested, here is something you could refer: 
1. https://github.com/biomejs/biome-zed (the code is not too much , I assume it is not hard to impl)
2. crates/oxc_language_server

Not sure if we need to maintain the editor extension ourselves.

**Comment by suxiaoshao** [comment]
I wrote an oxc extension for zed, the repository address is <https://github.com/suxiaoshao/zed-oxc>, and the pull request address is <https://github.com/zed-industries/extensions/pull/1430>. 
I have a question now: Are the command line parameters for oxc_language_server consistent with those of oxlint? Adding ‘–jsx-a11y-plugin’ doesn’t seem to work, and I can’t configure plugins in .oxlintrc.json. What should I do? @IWANABETHATGUY

**Reply by IWANABETHATGUY** [reply]
Nice, but I probably don't have time to have a look until Oct

**Comment by suxiaoshao** [comment]
I have developed the oxc extension and published it to the Zed editor's extension market. The repository address is <https://github.com/suxiaoshao/zed-oxc>. @Raynos, you can let your colleague who uses Zed try it out. However, there are still some issues; it cannot specify which plugins to use like the oxlint command line does.

**Reply by Boshen** [reply]
@suxiaoshao Feel free to transfer it into the `oxc-project` organization so we can maintain it together with more people, and link this in the oxc.rs website.

**Reply by suxiaoshao** [reply]
Okay, should I directly transfer this repository to the oxc-project organization? Does it need to be renamed to oxc-zed?
@Boshen

**Reply by Boshen** [reply]
> Okay, should I directly transfer this repository to the oxc-project organization? 

> Does it need to be renamed to oxc-zed?

Yes and yes. Thank you.

**Reply by suxiaoshao** [reply]
> > Okay, should I directly transfer this repository to the oxc-project organization?
> 
> > Does it need to be renamed to oxc-zed?
> 
> Yes and yes. Thank you.

GitHub tells me "You don’t have the permission to create public repositories on oxc-project.

**Reply by Boshen** [reply]
Add me to the repo with write permission, I can try and move it.

**Reply by suxiaoshao** [reply]
> Add me to the repo with write permission, I can try and move it.

I have already invited you.
